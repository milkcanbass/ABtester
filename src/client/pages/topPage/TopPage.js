import React from 'react';
import './TopPage.styles.scss';
import TopComponent from '../../components/topComponent/topComponent';
import PictureSection from '../../components/picturesSection/picturesSection';

const TopPage = () => (
  <>
    <TopComponent />
    {/* <PictureSection /> */}
  </>
);
export default TopPage;
