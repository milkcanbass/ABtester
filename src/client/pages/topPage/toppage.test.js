import { shallow } from 'enzyme';
import React from 'react';
import TopPage from './TopPage';

it('call', () => {
  expect.assertions();
  expect('hello').toEqual('hello');
});

describe('TopPageSnapshot', () => {
  it("should render correctly in 'debug' mode", () => {
    const component = shallow(<TopPage debug />);
    expect(component).toMatchSnapshot();
  });
});
