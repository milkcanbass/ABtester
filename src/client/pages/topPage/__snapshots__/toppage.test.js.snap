// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopPageSnapshot should render correctly in 'debug' mode 1`] = `
<Fragment>
  <TopComponent />
  <Connect(PicturesSection) />
</Fragment>
`;
